grails-app/domain/com/careerfly/organisation/College.groovy

```diff
@@ -0,0 +1,9 @@
+package com.careerfly.organisation
+
+class College extends Organisation {
+    Date dateCreated
```

Comment:
You should look for code repetition. These two fields `dateCreated` and `lastUpdated` are being duplicated in all 3 domains which extends `Organisation`. @akshitagangrade What could be the alternative?